### ISSUE TITLE
Improvement: Add time between broadcasts

### ISSUE THREAD

**Issue by WarLord211** [rkusa/DATIS#101]
I would be cool if the times between broadcasts were changeable.
ATIS etc are good but the Carrier broadcasts are too short.

the "99" transmissions came on the same Frequency as the normal carrier transmissions, but not that often.

**Comment by rkusa** [comment]
Good idea. Since I am a bit hesitant to add more options to the current string-based way of setting up options, I'll probably postpone this until I've added a Lua-based API to create/manage stations. This should make it easier to add as many options as we want.
